gallery-so/gallery
# [GAL-4257] "Connect Wallet" Upsell banner for email-only users [web]

### Summary of Changes

- Added upsell banner if the user don't any wallet connected
- Added copywriting to the sign in wallet modal


This backend PR need to merge first - https://github.com/gallery-so/go-gallery/pull/1187

### Demo or Before/After Pics

https://github.com/gallery-so/gallery/assets/4480258/2e00e7af-2cca-411f-8911-3af656ca9b4b

**Mobile View**

<img width="449" alt="CleanShot 2023-09-19 at 17 57 47@2x" src="https://github.com/gallery-so/gallery/assets/4480258/f5012864-cde1-4831-829d-4b361e41580b">

<img width="440" alt="CleanShot 2023-09-19 at 17 57 52@2x" src="https://github.com/gallery-so/gallery/assets/4480258/9cd702f4-fb3a-4039-bda7-1b5f1994e307">

- If this is a new feature, include screenshots or recordings of the feature in action.
- If this PR makes visual changes, include before and after screenshots. You can type `/table` to quickly come up with a before/after table.

### Edge Cases

1. Mobile view
2. Login with an user that has wallet connected

### Testing Steps

1. Login with an user without any wallet connected
2. You should see the blue banner
3. Close it will update the user experience flag

### Checklist

Please make sure to review and check all of the following:
